app/controllers/Healthcheck.scala

```diff
@@ -1,15 +1,22 @@
 package controllers
 
+import com.typesafe.scalalogging.LazyLogging
 import play.api.mvc._
 import services.TouchpointBackend
+import play.api.libs.concurrent.Execution.Implicits._
 
-object Healthcheck extends Controller {
+object Healthcheck extends Controller with LazyLogging{
 
-  def index = Action {
-    Cached(1) {
-      if (TouchpointBackend.Normal.zuoraService.products.nonEmpty) Ok("OK")
-      else ServiceUnavailable("Service Unavailable")
-    }
+  def index = Action.async {
+      (for {
```

Comment:
Minor detail, but should we create `isReady` methods in services, and maybe call these methods while looping over those services? It's just that health check might not be the ideal place to determine what it means for each service to be ready